### PR TITLE
add PullRequestController

### LINKED ISSUE #25: Afficher le détails d'Une PR

### PR DESCRIPTION
**Pull request by AxelR972**
partial PR for the PullRequestController and this function show

### CODE REVIEW

**Review by zakaria6907 — DISMISSED** [review]

File: src/Controller/PullRequestController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\PullRequestRepository;
+
+class PullRequestController extends AbstractController
+{
+    #[Route('/pull/request', name: 'app_pull_request')]
```
Comment (Dldvlpr) [inline src/Controller/PullRequestController.php:12]: this road name is different from that of line 20.
/pull/request and /pullRequest

they should have the same structure.

File: templates/pull_request/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<div class="example-wrapper">
+    <h1>Hello {{ controller_name }}! ✅</h1>
+
+    This friendly message is coming from:
+    <ul>
+        <li>Your controller at <code><a href="{{ 'C:/Users/axel_/Formation Dev Web/PR-tracker/WCS_Pull_Request_Tracker/src/Controller/PullRequestController.php'|file_link(0) }}">src/Controller/PullRequestController.php</a></code></li>
+        <li>Your template at <code><a href="{{ 'C:/Users/axel_/Formation Dev Web/PR-tracker/WCS_Pull_Request_Tracker/templates/pull_request/index.html.twig'|file_link(0) }}">templates/pull_request/index.html.twig</a></code></li>
+    </ul>
+</div>
+{% endblock %}
```
Comment (Dldvlpr) [inline templates/pull_request/index.html.twig:20]: It's generated code but we don't need it

File: templates/pull_request/show.html.twig
Comment (Dldvlpr) [inline templates/pull_request/show.html.twig:1]: empty file ?

**Review by Dldvlpr — CHANGES_REQUESTED** [review]

**Review by AxelR972 — COMMENTED** [review]
partial PR

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**